### DISCUSSION TITLE
Custom ffmpeg build

### DISCUSSION THREAD

**Discussion by drejkus** [blakeblackshear/frigate#5671] [General]
Hi, 
I managed to compile ffmpeg for my rk3399 board (RockPro64) with hwaccel support, but I'm having trouble making it run in Frigate. It works without hwaccel. As soon as I add:

```
ffmpeg:
  hwaccel_args: dri
```
it throws errors:
```
ffmpeg.okno.detect ERROR : Guessed Channel Layout for Input Stream #0.1 : mono
ffmpeg.okno.detect ERROR : [NULL @ 0xaaaac1d7e2e0] Unable to find a suitable output format for 'dri'
```
Am I missing anything that should be added as ffmpeg option?
Thank you.

**Comment by NickM-27** [comment]
Most likely it needs to be at least `-c:v dri` but really that's a question for wherever you compiled it.

**Reply by drejkus** [reply]
Thanks for superfast reply.
`-c:v dri` says `Unknown decoder 'dri'`. I compiled ffmpeg with instructions from [here](https://forum.clockworkpi.com/t/a06-mainline-hardware-video-decoding-tested-working/9239). 

There is an example of use with hwaccel and without:
```
without hw dec: ffmpeg -i input.mp4 -f null - -benchmark
with hw dec: ffmpeg -hwaccel drm -i input.mp4 -f null - -benchmark
```
which works for me. So i presumed I should put only `drm` to `hwaccel_args`

**Reply by drejkus** [reply]
Jeez, I made a typo, it should be `hwaccel_args: drm`, not `hwaccel_args: dri`. But it behaves the same.

**Reply by NickM-27** [reply]
In that case the args would be `-hwaccel drm`

**Reply by drejkus** [reply]
YES! Thank you! Such a stupid mistake...
